### PR TITLE
fix java11 build

### PR DESCRIPTION
**Pull request by JerryShea**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -203,30 +203,10 @@
                 </configuration>
             </plugin>
 
-	    <plugin>
+	        <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
-                <configuration>
```
Comment (JerryShea) [inline pom.xml:210]: errorprone plugin was causing odd errors in java 11

File: pom.xml [hunk 2]
```diff
@@ -302,6 +284,22 @@
             <properties>
                 <makefile>Makefile11</makefile>
             </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <configuration>
+                            <compilerArgs>
```
Comment (JerryShea) [inline pom.xml:290]: generate JNI .h here instead of Makefile

**Review by nicktindall — APPROVED** [review]

File: src/main/c/Makefile11 [hunk 3]
```diff
@@ -43,9 +43,5 @@ all: $(TARGET)
 $(TARGET): $(JNI_HEADERS) $(JNI_SOURCES)
```
Comment (nicktindall) [inline src/main/c/Makefile11]: Do we need to remove `JNI_HEADERS` from here as well?

Comment (JerryShea) [reply src/main/c/Makefile11]: done - thx!

**Pull request merged**